ray-operator/test/e2e/raycluster_autoscaler_test.go

```diff
@@ -0,0 +1,77 @@
+package e2e
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	rayv1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1"
+	"github.com/ray-project/kuberay/ray-operator/controllers/ray/common"
+	rayv1ac "github.com/ray-project/kuberay/ray-operator/pkg/client/applyconfiguration/ray/v1"
+	. "github.com/ray-project/kuberay/ray-operator/test/support"
+)
+
+func TestRayClusterAutoscaler(t *testing.T) {
```

Comment:
A high level question: should we have all the testing logic inside python? like you can have a python script:

```
ray.init()
actor1 = Actor.remote()
assert ray.nodes() == 2
actor2 = Actor.remote()
assert ray.nodes() == 3
```

instead of doing the assertion in go?

Reply:
Got it makes sense.